### PR TITLE
 added function when one faq opens the other closes

### LINKED ISSUE #524: When another item is clicked on the FAQ previous opened item won't close.
**Summary:**

When another item is clicked on the FAQ previous opened item won't close. We need only one (or zero) expanded group at a time.

**Device and Android version:**

Device Real Me 2
Android Version 8.1.0

**Screenshots:**

Screenshot_2019-03-27-14-59-26-39

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #524
Added function when one FAQ opens, the other FAQ closes automatically

- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java [hunk 1]
```diff
@@ -84,6 +87,20 @@ public void initObjects() {
 
     }
 
+    public void initExpand() {
+        expandableListView.setOnGroupExpandListener(
+                new ExpandableListView.OnGroupExpandListener() {
+                    int expand_pos = -1;
+                    @Override
+                    public void onGroupExpand(int groupPosition) {
+                        if (expand_pos >= 0 && expand_pos != groupPosition) {
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: If the user clicks on the expanded item, it should collapse. Does your code restricts that?

Comment (Bhavnaharitsa) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: ezgif com-video-to-gif
 No it does not. It works fine

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java [hunk 2]
```diff
@@ -84,6 +87,20 @@ public void initObjects() {
 
     }
 
+    public void initExpand() {
+        expandableListView.setOnGroupExpandListener(
+                new ExpandableListView.OnGroupExpandListener() {
+                    int expand_pos = -1;
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.java]: Give this variable a better name, in camel case.

**Comment by Bhavnaharitsa** [comment]
Sir, I am having a tough time squashing. I shall learn soon until then please bare with me.
For now, closing this pr

**Comment by laxyapahuja** [comment]
> Sir, I am having a tough time squashing. I shall learn soon until then please bare with me.
> For now, closing this pr

hey @Bhavnaharitsa, i think [this](https://gist.github.com/laxyapahuja/e30921a9478bdf3a9e166a023ce1882c) will help you :)

**Pull request closed without merging**